pkg/utils/utils_test.go

```diff
@@ -81,3 +81,31 @@ func TestTrimTrailingNewline(t *testing.T) {
 		assert.EqualValues(t, s.expected, TrimTrailingNewline(s.str))
 	}
 }
+
+func TestNormalizeLinefeeds(t *testing.T) {
+	type scenario struct {
+		byteArray []byte
+		expected  []byte
+	}
+	var scenarios = []scenario{
+		{
+			// \r\n
+			[]byte{97, 115, 100, 102, 13, 10},
```

Comment:
we should add a test for when we have blah\r\n\blah to ensure we still end up with a newline between the words